scipy/stats/morestats.py

```diff
@@ -78,11 +79,20 @@ def bayes_mvs(data, alpha=0.90):
     standard-deviation from data", http://hdl.handle.net/1877/438, 2006.
 
     """
-    res = mvsdist(data)
+    m, v, s = mvsdist(data)
     if alpha >= 1 or alpha <= 0:
         raise ValueError("0 < alpha < 1 is required, but alpha=%s was given."
                          % alpha)
-    return tuple((x.mean(), x.interval(alpha)) for x in res)
+
+    mean = namedtuple('mean', ('estimate', 'minmax'))
```

Comment:
Capitalize? Also, `statistic` might be more consistent with the rest of the changes name compared to `estimate`